### PR TITLE
feat(hero): implement new hero background colour

### LINKED ISSUE #180: Remove Gradient from Hero component
The Hero component gradient needs to be flattened to a solid colour. The gradient is longer needed for any components and can be completely removed.

The colour may exist already, or may need adding to the palette.

Colour name: `--color-super-blue-dark` 

Colour hex: `#175EA1`

### PR DESCRIPTION
**Pull request by owensgit**
# Description

Implements new flat colour for the hero background.

Also deprecates the following CSS vars: `--hero-bg-image` and `--gradient-super-blue`. I will add these to the **[Deprecations & upcoming breaking changes issue](https://github.com/Legal-and-General/canopy/issues/196)** when this PR is merged. I've marked them with a CSS comment: `/* deprecated */` - let me know if you recommend a better way of marking it.

Fixes #180

Storybook link: (once netlify has deployed link provide a link to the component)

Screenshots:

Screenshot 2021-02-04 at 13 00 57

Screenshot 2021-02-04 at 13 01 07

# Checklist:

- [x] The commit messages follow the [convention for this project](./blob/master/docs/CONTRIBUTING.md#conventional-commits)
- [ ] I have provided an adequate amount of test coverage
- [ ] I have added the functionality to the [test app](./blob/master/docs/CONTRIBUTING.md#build-test-application)
- [ ] I have provided a story in storybook to document the changes
- [ ] I have provided documentation in the notes section of the story
- [ ] I have added any new public feature modules to public-api.ts
- [ ] I have [linked the new component](<(https://github.com/Legal-and-General/canopy/blob/master/docs/CONTRIBUTING.md#invision-dsm)>) to adobe DSM (if appropriate)

### CODE REVIEW

**Review by thatguynamedandy — DISMISSED** [review]
Nice one @owensgit I think reducing the amount of breaking changes is good practice. If nothing else it's a good experiment.

**Comment by thatguynamedandy** [comment]
One minor thought is adding `@` to the beginning of deprecated (e.g. `@deprecated`)  just to make it stand out from other uses of that word (not that there are any), and it also aligns to JSDoc syntax, but at this stage it wouldn't bring much real benefit

**Review by elenagarrone — APPROVED** [review]

**Comment by owensgit** [comment]
@elenagarrone  @thatguynamedandy  thanks for your reviews, I've updated the deprecated comment to `/* @deprecated */`, please take another look when you can.

**Review by thatguynamedandy — APPROVED** [review]

File: projects/canopy/src/lib/hero/hero-header/hero-header.component.scss [hunk 1]
```diff
@@ -10,7 +10,7 @@
   bottom: 0;
   left: 0;
   right: 0;
-  background-color: var(--hero-bg-color);
+  background-color: var(--hero-overlay-bg-color);
   opacity: var(--hero-bg-alpha);
```
Comment (paul-request) [inline projects/canopy/src/lib/hero/hero-header/hero-header.component.scss:14]: Doesn't that mean that any content inside the hero is also going to be slightly opaque? If so, wouldn't an rgba background colour be better?

**Review by paul-request — APPROVED** [review]

File: projects/canopy/src/lib/hero/hero-header/hero-header.component.scss (hunk 1 above)
Comment (paul-request) [reply projects/canopy/src/lib/hero/hero-header/hero-header.component.scss:14]: Ignore me, its the overlay isn't it

**Pull request merged**